templates/zerver/help/mute-a-stream.md

```diff
@@ -0,0 +1,23 @@
+### Mute a stream
+
+Muting streams to reduce notification clutter is easy. Simply hover your mouse
+over the stream in the streams list on the left of the chat, press the down
+chevron button
+
+![streams arrow](static/images/help/streams-arrow.png)
+
+to bring up the stream popover menu and press **Mute the stream**. The muted
+stream should be greyed out in the streams list.
```

Comment:
Wording is a bit weird, try "After clicking **Mute the stream**, the muted stream will turn gray in the streams list" because it's something that happens after you do an action.